pkg/dynamicaccess/act.go

```diff
@@ -0,0 +1,55 @@
+package dynamicaccess
+
+import (
+	"encoding/hex"
+
+	"github.com/ethersphere/bee/pkg/manifest"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type Act interface {
+	Add(lookupKey []byte, encryptedAccessKey []byte) Act
+	Get(lookupKey []byte) []byte
```

Comment:
If there is "Get", then usually we are writing "Put" as a counterpart.
If there is "Add" we are writing "Lookup" as a counterpart.
// Viktor

Reply:
`lookupkey` should be `key`, `encryptedAccessKey` should be value, and it should give back an `error` as well.
If it's a mutable object, it should only give back error (it is changing the object itself).
Both should have an `error` in the return value.
// Viktor

Reply:
A doc comment should be before the Act.
// Viktor

Reply:
PR: https://github.com/Solar-Punk-Ltd/bee/pull/13

Reply:
key, value, error, doc
PR: https://github.com/Solar-Punk-Ltd/bee/pull/14

Reply:
IF has been refactored accordingly